RELEASE.md

```diff
@@ -0,0 +1,28 @@
+## Quality gates
+
+| Risk level                    | Edge                            | Beta                         | Candidate                                                                                    | Stable                                                                                                                                            |
+|-------------------------------|---------------------------------|------------------------------|----------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| Meaning                       | Bleeding edge developer version | Most new features stabilized | Feature-ready, currently in testing                                                          | Well-tested, production-ready                                                                                                                     |
+| Preconditions (quality gates) | Code review, CI                 |                              | Load tests                                                                                   |                                                                                                                                                   |
+| Timing                        | Every merge to main             |                              | Charm reaches a state of feature completion with respect to the next planned stable release. | In consultation with product manager and engineering manager when the candidate revision has been well tested and is deemed ready for production. |
+| Release process               | Automatic (on merge)            | Manual                       | Manual                                                                                       | Manual                                                                                                                                            |
+
+
+## Publishing
+```shell
+./render_bundle.py bundle.yaml --channel=edge
+charmcraft pack
+charmcraft upload cos-lite.zip
+charmcraft release cos-lite --channel=edge --revision=N
+```
+
+### Promoting a revision to a lower risk level
+```shell
+charmcraft release cos-lite --channel=beta --revision=N
```

Comment:
we probably should not be doing (or recommending) this, as it will mean that beta would point at the edge charms, wouldn't it?

Reply:
The problem here is that the rendered template will have a channel target for each charm pointing at edge. So for bundles, we'd need to regenerate the yaml file for each channel.

Reply:
ie. https://github.com/canonical/cos-lite-bundle/blob/bdd28760f4abe7c07970ec148c71741f29ec378b/bundle.yaml.j2#L16 this will point at the wrong channel if we promote an edge bundle to beta for instance

Reply:
I thought for bundles the `--channel` option of `juju deploy` determined what channel all the charms came from. Is this not the case?

Reply:
Oh yeah, you're absolutely right! Will fix in next commit.
@dstathis I'm afraid that is not the case.

Reply:
@dstathis No, sorry, it determines which channel to pull the bundle manifest from 😅